src/standalone/Selector/MultiSelect.tsx

```diff
@@ -166,10 +166,8 @@ const MultiSelect = <Data extends MultiSelectorData>(
 	);
 };
 
-const StylesMultiSelect = withStyles(styles)(React.memo(MultiSelect)) as <
-	Data extends MultiSelectorData
->(
-	props: GenericWithStyles<MultiSelectProps<Data> & WithStyles>
+const StylesMultiSelect = withStyles(styles)(React.memo(MultiSelect)) as (
```

Comment:
`export default React.memo(MultiSelect)` is enough. withStyles shouldn't be used for functional components